question-servers/calculation-worker.js

```diff
@@ -0,0 +1,202 @@
+// @ts-check
+// This is meant to be invoked from `./question-servers/calculation.js`. It
+// serves to isolate code from the main process that's handling requests.
```

Comment:
Does this comment need updating? We are actually invoking this from the python zygote now?

Reply:
Yep! Good call; I'll change this.

Reply:
Resolved in d1a147c.